tests/test_modalities/test_ophys/test_tools_roiextractors.py

```diff
@@ -536,6 +536,26 @@ def test_rejected_roi_ids(self, rejected_list, expected_rejected_roi_ids):
         plane_segmentation_accepted_roi_ids = plane_segmentation["Accepted"].data
         assert_array_equal(plane_segmentation_accepted_roi_ids, accepted_roi_ids)
 
+    def test_add_plane_segmentation_with_quality_metrics(self):
+        """Test that available quality metrics in the segmentation extractor are added as columns to the PlaneSegmentation when available."""
```

Comment:
This looks fishy, 
if & is intersection I think this test will always pass. or am I wrong?

 Anyway, for these test here we have absolute control. Can you add some specific properties to the segmentation extractor here and test:
1) that they are added
2) their values match what we just added.

So just create a segmentation extractor, add properties with values that you control of various dtypes and check that they are written correctly.